src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
I mean lightweight computationally. We are adding up miliseconds to startup by each new feature that we add, and that's how heavy programs get born ;)